Sources/DotLottie/Private/DotLottieManager.swift

```diff
@@ -0,0 +1,194 @@
+        return try getPlaybackSettings(animationId: self.currentAnimationId)
+    }
+
+    /// Loads the next animation from the dotLottie.
+    /// - Returns: ManifestAnimationModel object containing the playback settings of the n+1 animation. Constructed from information inside the manifest file.
+    public func nextAnimation() throws -> ManifestAnimationModel {
+        if let manifest = self.manifest {
+            
+            let index = manifest.animations.firstIndex { animation in
+                if animation.id == self.currentAnimationId {
+                    return true
+                } else {
+                    return false
+                }
+            }
+            
+            if var checkedIndex = index  {
+                if checkedIndex < manifest.animations.count - 1 {
+                    checkedIndex += 1
+                    
+                    self.currentAnimationId = manifest.animations[checkedIndex].id
+                }
+                
+                return manifest.animations[checkedIndex]
+            }
+        }
+        
+        throw DotLottieManagerErrors.missingAnimationInsideManifest
+    }
+    
+    /// Loads the previous animation from the dotLottie.
+    /// - Returns: ManifestAnimationModel object containing the playback settings of the n-1 animation. Constructed from information inside the manifest file.
+    public func prevAnimation() throws -> ManifestAnimationModel {
+        if let manifest = self.manifest {
+            
+            let index = manifest.animations.firstIndex { animation in
+                if animation.id == self.currentAnimationId {
+                    return true
+                } else {
+                    return false
+                }
+            }
+            
+            if var checkedIndex = index  {
+                if checkedIndex > 0 {
+                    checkedIndex -= 1
+                    
+                    self.currentAnimationId = manifest.animations[checkedIndex].id
+                }
+
+                return manifest.animations[checkedIndex]
+            }
+        }
+        
+        throw DotLottieManagerErrors.missingAnimationInsideManifest
+    }
+    
+    
+    /// Set the current playing animation.
+    /// - Parameter animationId: Desired animation to play.
+    public func setActiveAnimation(animationId: String) throws {
+        if !self.containsAnimation(animationId: animationId) {
+            throw DotLottieManagerErrors.missingAnimationInsideManifest
+        }
+        
+        self.currentAnimationId = animationId
+    }
+    
+    
+    /// Returns the path on disk to the desired animation.
+    /// - Parameter animationId: Animation to search for.
+    /// - Returns: URL to a path on disk where the animation data in .json format is located.
+    public func getAnimationPath(_ animationId: String) throws -> URL {
+        if !self.containsAnimation(animationId: animationId) {
```

Comment:
I guess this works, but another way to have this would be:

guard let filePath = filePaths[animationId] else {
   throw DotLottieManagerErrors.missingAnimationInsideManifest
}

return filePath